openghg/exceptions/__init__.py

```diff
@@ -0,0 +1,8 @@
+
+
+class OpenGHGError(Exception):
```

Comment:
These could be moved to `openghg.types` as we've already got some custom exceptions there.